src/analyzer/plugins/analyzerqueenmarybeats.h

```diff
@@ -49,7 +49,10 @@ class AnalyzerQueenMaryBeats : public AnalyzerBeatsPlugin {
     std::unique_ptr<DetectionFunction> m_pDetectionFunction;
     DownmixAndOverlapHelper m_helper;
     int m_iSampleRate;
-    std::vector<double> m_detectionResults;
+    int m_windowSize;
+    int m_stepSize;
+    std::vector<double>
+            m_detectionResults;
```

Comment:
Newline typo?

Reply:
Btw are you using our pre-commit hooks? This should fix these style issues automatically when committing.
https://www.mixxx.org/wiki/doku.php/using_git#set_up_automatic_code_checking

Reply:
Not using the pre-commit hooks, but using clang-format. This line break was added automatically by VS Code so I leave it, should remove?

Reply:
Then something might be wrong with your .clang-format setup? I can confirm that VS Code works as expected and does not insert unneeded line breaks.

Reply:
I Will double-check my settings or I might have accidentally added without noticing after all. Either way will setup the pre-commit hooks too.